packages/remix-netlify-edge/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "@remix-run/netlify-edge",
+  "description": "Netlify Edge platform abstractions for Remix",
+  "version": "1.4.3",
+  "license": "MIT",
+  "types": "./index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/remix-run/remix",
+    "directory": "packages/remix-netlify-edge"
+  },
+  "bugs": {
+    "url": "https://github.com/remix-run/remix/issues"
+  },
+  "dependencies": {
+    "@remix-run/server-runtime": "1.4.3"
```

Now that we have `@remix-run/deno`, we should use that package instead

```suggestion
    "@remix-run/deno": "1.4.3"
```